website/src/app/kb/authenticate/okta/readme.mdx

```diff
@@ -3,31 +3,28 @@ import PlanBadge from "@/components/PlanBadge";
 import Image from "next/image";
 import Link from "next/link";
 
-<PlanBadge plans={["enterprise"]}>
+<PlanBadge plans={["starter", "team", "enterprise"]}>
 
-# SSO + Sync with Okta
+# SSO with Okta
 
 </PlanBadge>
 
 Firezone integrates with Okta using a custom connector that supports both
 authentication and directory sync. Use this guide if you're looking to setup SSO
-with Okta for your Firezone Enterprise account and want to automatically sync
-users and groups from Okta to Firezone.
+with Okta for your Firezone Enterprise account and optionally sync users and
+groups from Okta to Firezone.
```

Comment:
Similar thing here as with Entra ID, it kind of sounds like this guide is only for enterprise accounts.